Mathlib/Algebra/FreeMonoid/Basic.lean

```diff
@@ -238,6 +237,11 @@ def lift : (α → M) ≃ (FreeMonoid α →* M)
 #align free_monoid.lift FreeMonoid.lift
 #align free_add_monoid.lift FreeAddMonoid.lift
 
+-- porting note: new
```

Comment:
I think we should maybe distinguish "added to make the port work" from "added after the port was complete"

Reply:
How?

Reply:
I think it doesn't matter too much. A simple `git blame` can tell whether it was added in a porting PR or in a "regular" PR.

Reply:
Only if we adopt some convention for marking in the git commit message; this isn't purely hypothetical, I'd like to be able to display mathlib4 porting commits on the port status page, and need a way to identify them

Reply:
How about starting porting commit titles by "port:" instead of "feat:"?